pkg/disruptors/pod.go

```diff
@@ -50,6 +56,50 @@ type PodAttributes struct {
 	Labels map[string]string
 }
 
+// NamespaceOrDefault returns the configured namespace for this selector, and the name of the default namespace if it
+// is not configured.
+func (p PodSelector) NamespaceOrDefault() string {
+	if p.Namespace != "" {
+		return p.Namespace
+	}
+
+	return metav1.NamespaceDefault
+}
+
+// String returns a human-readable explanation of the pods matched by a PodSelector.
+func (p PodSelector) String() string {
+	var str string
+
+	if len(p.Select.Labels) == 0 && len(p.Exclude.Labels) == 0 {
+		str = "all pods"
+	} else {
+		str = "pods "
+		if len(p.Select.Labels) != 0 {
+			str += "including("
+			for k, v := range p.Select.Labels {
+				str += fmt.Sprintf("%s=%s, ", k, v)
+			}
+			str = strings.TrimSuffix(str, ", ")
+			str += "), "
+		}
+
+		if len(p.Exclude.Labels) != 0 {
+			str += "excluding("
+			for k, v := range p.Exclude.Labels {
+				str += fmt.Sprintf("%s=%s, ", k, v)
+			}
+			str = strings.TrimSuffix(str, ", ")
+			str += "), "
+		}
```

Comment:
These two blocks seem identical except for the prefix ("excluding", "including"). Maybe it worth using a small function that returns the labels enclosed in the prefix or null is len if `0`

Reply:
I didn't do this initially because I was afraid that the function declaration boilerplate was going to be larger than just duplicating, but after trying it I actaully think it does look nicer. LMK what think: https://github.com/grafana/xk6-disruptor/pull/267/commits/45f971dcbcf6e902635aa73a58410d1c2b0a2852

Reply:
Nice. Looks cleaner.